tekton/README.md

```diff
@@ -31,25 +31,55 @@ This cluster
 
 To make a new release:
 
+1. [Create draft release](#create-draft-release) in GitHub with release notes
 1. (Optionally) [Apply the latest versions of the Tasks + Pipelines](#setup)
-2. (If you haven't already)
+1. (If you haven't already)
    [Install `tkn`](https://github.com/tektoncd/cli#installing-tkn)
-3. [Run the Pipeline](#run-the-pipeline)
-4. Create the new tag and release in GitHub
+1. [Run the Pipeline](#run-the-pipeline)
+1. Create the new tag and release in GitHub
    ([see one of way of doing that here](https://github.com/tektoncd/pipeline/issues/530#issuecomment-477409459)).
    _TODO(tektoncd/pipeline#530): Automate as much of this as possible with
    Tekton._
-5. Add an entry to [the README](../README.md) at `HEAD` for docs and examples
+1. Add an entry to [the README](../README.md) at `HEAD` for docs and examples
    for the new release ([README.md#read-the-docs](README.md#read-the-docs)).
-6. Update the new release in GitHub with the same links to the docs and
+1. Update the new release in GitHub with the same links to the docs and
    examples, see
    [v0.1.0](https://github.com/tektoncd/pipeline/releases/tag/v0.1.0) for
    example.
 
+### Create draft release
+
+The Task `create-draft-triggers-release` calculates the list of PRs merged between the previous release and a specified revision. It also builds a list of authors and uses PRs and authors to build a draft new release in GitHub.
+
+Running this Task multiple times will create multiple drafts; old drafts have to be pruned manually when needed.
```

Comment:
Would be really nice to fix this someday!

Reply:
I don't think this is necessary bad behavior. I see it as a safety feature that your old draft will not be overwritten. For example, if I've created a draft release, organized the PRs, and added some additional text, I wouldn't want that draft to be overwritten by running the Task.